### DISCUSSION TITLE
Extract without headers and footers

### DISCUSSION THREAD

**Discussion by lawrencenika** [jsvine/pdfplumber#1028] [Ask for help with specific PDFs]
I have a PDF, is it possible to extract_text() without headers or footers?
[Life_Cycle_Assessment_of_Cow_Tanned_Leather_Produc.pdf](https://github.com/jsvine/pdfplumber/files/13174999/Life_Cycle_Assessment_of_Cow_Tanned_Leather_Produc.pdf)

**Comment by jsvine** [comment]
Please see this related discussion, and let us know if that answers your question: https://github.com/jsvine/pdfplumber/discussions/968
